hexrd/material.py

```diff
@@ -475,6 +495,16 @@ def _readHDFxtal(self, fhdf=DFLT_NAME, xtal=DFLT_NAME):
         else:
             self.stiffness = numpy.zeros([6, 6])
 
+        if('dmin' in gid):
+            # if dmin is present in the HDF5 file, then use that
+            dmin = numpy.asscalar(numpy.array(gid.get('dmin'), \
+                                    dtype = numpy.float64))
```

Comment:
`numpy.asscalar()` is [deprecated since version 1.16](https://numpy.org/doc/stable/reference/generated/numpy.asscalar.html). Looks like `.item()` is recommended instead:
```python
numpy.array(gid['dmin'], dtype=numpy.float64).item()
```